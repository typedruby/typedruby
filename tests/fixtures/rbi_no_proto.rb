def typed => nil
  untyped(0)
  no_arg
end
