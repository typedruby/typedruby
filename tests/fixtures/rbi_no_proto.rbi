def untyped(x)
end

def no_arg
end

def partial(x) => nil
end
