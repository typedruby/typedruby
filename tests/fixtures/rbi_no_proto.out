error: Definitions in .rbi files must contain type annotations

        @ __ROOT__/tests/fixtures/rbi_no_proto.rbi:1
      1 |  def untyped(x)
                      ^^^ here

error: Definitions in .rbi files must contain type annotations

        @ __ROOT__/tests/fixtures/rbi_no_proto.rbi:4
      4 |  def no_arg
               ^^^^^^ here
